inlong-sort/sort-formats/format-base/src/main/java/org/apache/inlong/sort/formats/base/TableFormatConstants.java

```diff
@@ -38,6 +39,11 @@ public class TableFormatConstants {
     public static final char DEFAULT_DELIMITER = ',';
     public static final char DEFAULT_ENTRY_DELIMITER = '&';
     public static final char DEFAULT_KV_DELIMITER = '=';
+    public static final Character DEFAULT_LINE_DELIMITER = null;
+    public static final Character DEFAULT_ESCAPE_CHARACTER = null;
+    public static final Character DEFAULT_QUOTE_CHARACTER = null;
+    public static final String DEFAULT_NULL_LITERAL = null;
     public static final boolean DEFAULT_IGNORE_ERRORS = false;
     public static final String DEFAULT_CHARSET = "UTF-8";
+    public static final int DEFAULT_BYTE_ARRAY_STREAM_LENGTH = 2048;
```

Comment:
not used, suggest use idea basic code check plugin to check again

Reply:
delete